packages/formation/sass/base/_va.scss

```diff
@@ -97,6 +97,18 @@ a {
   }
 }
 
+// Use outline focus styling for links in the main content area
+// since box-shadow styling doesn't look good on wrapped links
+main [href],
+va-banner [href] {
+  &:focus {
+    outline: $focus-outline;
+    background: $color-white;
+    color: $color-link-default;
+    box-shadow: unset;
```

Comment:
The gold outline gives us color contrast on light backgrounds, and setting the focused element's background to white and giving it a darker text ensures that we'll have good contrast for links on dark backgrounds.